src/utils.ts

```diff
@@ -99,6 +99,23 @@ function display(value: unknown) {
   return typeof value === 'string' ? value : stringify(value);
 }
 
+export function stringifyDefined(input: unknown) {
+  if (input == null || typeof input !== 'object' || Array.isArray(input)) {
```

Comment:
```suggestion
  if (input === null || typeof input !== 'object' || Array.isArray(input)) {
```
is there a specific reason to go for "=="?

Reply:
Yes, `==` also catches `undefined` while `===` would not do it. On the other hand `typeof input !== 'object'` would also catch `undefined`